config_spec.toml

```
@@ -37,3 +37,9 @@ name = "lnd_macaroon_path"
 type = "std::path::PathBuf"
 doc = "Path to LND macaroon with open channel permission"
 optional = false
+
+[[param]]
+name = "localhost_ip"
+type = "std::net::Ipv4Addr"
+doc = "The local loopback address of your machine (usually 127.0.0.1 or 0.0.0.0 in the case of docker)"
```

```suggestion
name = "bind_ip"
type = "std::net::Ipv4Addr"
doc = "The interface from which this server should listen (usually 127.0.0.1 or 0.0.0.0 in the case of docker)"
```

localhost really just means `127.0.0.1` whereas this variable that can be anything you make it. Consider moving next to `bind_port`.